torchvision/prototype/transforms/functional/_meta.py

```diff
@@ -154,3 +154,23 @@ def convert_image_color_space_pil(
         return image
 
     return image.convert(new_mode)
+
+
+def convert_color_space(
+    inpt: Any, *, color_space: ColorSpace, old_color_space: Optional[ColorSpace] = None, copy: bool = True
+) -> Any:
+    if isinstance(inpt, Image):
+        return inpt.to_color_space(color_space, copy=copy)
+    if isinstance(inpt, _Feature):
+        return inpt
```

Comment:
I don't think the kernels should offer pass through functionality here. No other kernel does as far as I recall. Do we really need this?
```suggestion
```

Reply:
> No other kernel 

You mean _dispatcher_, right? This is not a kernel. 

Plus, all current dispatchers implement a pass through. The calls look like

https://github.com/pytorch/vision/blob/4db84b04838523594c7fa2250649492e0d41e150/torchvision/prototype/transforms/functional/_geometry.py#L51-L52

and the default implementation is

https://github.com/pytorch/vision/blob/4db84b04838523594c7fa2250649492e0d41e150/torchvision/prototype/features/_feature.py#L89-L90

I'm all for changing this default behavior, but so far this PR is consistent with the rest of the API.

Reply:
As discussed offline, Vasilis is right to remove this branch here. There are two places where a passthrough should be defined:

1. The transform knows that a specific type cannot be handled and thus doesn't pass it to the dispatcher in the first place.
2. The feature does not overwrite the dispatch methods which defaults to passthrough.

The dispatcher itself should not do this.